.ruby-version

```diff
@@ -1 +0,0 @@
-2.3.0
```

Comment:
Since we are supporting all the way down to 2.0, I didn't want to specify 2.3 for development.